src/silx/gui/plot/backends/glutils/GLPlotFrame.py

```diff
@@ -215,6 +216,8 @@ def getVerticesAndLabels(self):
         labels = []
         tickLabelsSize = [0., 0.]
 
+        font = qt.QApplication.instance().font()
```

Comment:
Using QApplication default for plot labels/title. This can be made configurable if needed